controllers/configurationpolicy_controller.go

```diff
@@ -1193,8 +1195,14 @@ func (r *ConfigurationPolicyReconciler) handleObjectTemplates(plc policyv1.Confi
 // checkRelatedAndUpdate checks the related objects field and triggers an update on the ConfigurationPolicy
 func (r *ConfigurationPolicyReconciler) checkRelatedAndUpdate(
 	plc policyv1.ConfigurationPolicy, related, oldRelated []policyv1.RelatedObject, sendEvent bool,
+	isTemplateParseErr ...bool,
 ) {
-	r.sortRelatedObjectsAndUpdate(&plc, related, oldRelated, r.EnableMetrics)
+	tempParseErr := false
+	if len(isTemplateParseErr) > 0 {
+		tempParseErr = isTemplateParseErr[0]
+	}
+
+	r.sortRelatedObjectsAndUpdate(&plc, related, oldRelated, r.EnableMetrics, tempParseErr)
```

Comment:
why I didn't use "variadic parameter", sortRelatedObjectsAndUpdate is only used here